Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -75,6 +76,25 @@ AttributedStringViewIssueDelegate {
         return AlertAction(AlertActionBuilder { $0.rootViewController = weakSelf?.viewController })
             .share([url], activities: [TUSafariActivity()]) { $0.popoverPresentationController?.sourceView = sender }
     }
+    
+    func deleteAction() -> UIAlertAction? {
+        guard object?.viewerCanDelete == true, let number = object?.number else { return nil }
+        
+        return AlertAction.delete({ [weak self] _ in
```

Comment:
Nit: Why the `({` instead of the ` {`? :)

Reply:
Copy and Pasta, will change